test/e2e/e2e_test.go

```diff
@@ -999,7 +999,7 @@ func TestDeschedulingInterval(t *testing.T) {
 }
 
 func waitForRCPodsRunning(ctx context.Context, t *testing.T, clientSet clientset.Interface, rc *v1.ReplicationController) {
-	if err := wait.PollImmediate(5*time.Second, 30*time.Second, func() (bool, error) {
+	if err := wait.PollImmediate(5*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
does it really take that long to bring up the e2e pods? what is causing the sudden slow down?

Reply:
hi, yes, we need it, recently the pull request always fail at pod pending, you can check as below
```
...
=== RUN   TestTopologySpreadConstraint/test-rc-topology-spread-soft-constraint
    e2e_topologyspreadconstraint_test.go:59: Creating RC test-rc-topology-spread-soft-constraint with 4 replicas
    e2e_test.go:1010: Waiting for 4 pods to be created, got 0 instead
    e2e_test.go:1015: Pod test-rc-topology-spread-soft-constraint-27t6z not running yet, is Pending instead
    e2e_test.go:1015: Pod test-rc-topology-spread-soft-constraint-27t6z not running yet, is Pending instead
    e2e_test.go:1015: Pod test-rc-topology-spread-soft-constraint-27t6z not running yet, is Pending instead
    e2e_test.go:1015: Pod test-rc-topology-spread-soft-constraint-27t6z not running yet, is Pending instead
    e2e_test.go:1015: Pod test-rc-topology-spread-soft-constraint-27t6z not running yet, is Pending instead
    e2e_test.go:1015: Pod test-rc-topology-spread-soft-constraint-27t6z not running yet, is Pending instead
    e2e_test.go:1015: Pod test-rc-topology-spread-soft-constraint-27t6z not running yet, is Pending instead
    e2e_test.go:1021: Error waiting for pods running: timed out waiting for the condition
    e2e_test.go:1064: Waiting for test-rc-topology-spread-soft-constraint RC pods to disappear, still 4 remaining
    e2e_test.go:1064: Waiting for test-rc-topology-spread-soft-constraint RC pods to disappear, still 0 remaining
--- FAIL: TestTopologySpreadConstraint (72.30s)
```

Reply:
It's just such a big bump.

0-30 seconds -> 7 checks (as shown in the logs)
0-300 seconds -> 61 checks

Reply:
yes, i get your point, i extend the timeout to 5 minutes it is for  pull request always fail at pod pending recently, and extend the timeout now the pull request will not fail at pod pending again and again. what is your suggestion for this part to avoid the pod pending and type /retest again and again?

Reply:
I think the question is why 5 minutes rather than 2 or 3? Do we have an average startup time for the pods to compare against? I see the other test in this PR times out at 2 minutes

Reply:
sorry for i just get it now. why 5 minutes rather than 2 or 3 is a very good question.
i need to finger out the average startup time first.